docs/AQL/aql.g

```diff
@@ -37,13 +37,16 @@ public void displayRecognitionError(String[] tokenNames, RecognitionException e)
 //<Query> ::= <Select> <From>
 //           | <Select> <From> <Where>
 //            | <Select> <From> <OrderBy>   ! is this allowed?
+//            | <Select> <From> <OrderBy> <Limit>
 //            | <Select> <From> <Where> <OrderBy>
-query	:	select from where? orderBy? ';'!? EOF!;
+//            | <Select> <From> <Where> <OrderBy> <Limit>
+query	:	select from where? orderBy? limit? ';'!? EOF!;
 
 //<Select> ::= 'SELECT' <SelectExpr>
-//         | 'SELECT' <TOP> <SelectExpr>
+//         | 'SELECT' <TOP> <SelectExpr>   deprecated!
 select	:	SELECT top? selectExpr -> ^(SELECT top? selectExpr);
 
+// (deprecated)
```

Comment:
To be correct with respect to versions, we probably should leave the TOP-relate bit of the grammar in (of course with new comments indicating it is deprecated), since removing it should really only occur in v2 of AQL. If we can get agreement from all implems to the contrary then fine.

Reply:
It is still part of grammar, I just added a comment line.